src/services/ml5-service.ts

```diff
@@ -22,7 +24,7 @@ const classifyImage = async ({imageUrl}) => {
   log('classifying...');
   await requireMl5();
```

Comment:
Cool. I'll make the change here.

Reply:
Similar to the note above: Will the "get property off of window" method not work in a node environment?

Reply:
issue addressed in 9e6a324

Reply:
There is no `window` in Node, the nearest analog is `self`. But `ml5` in particular requires DOM apis, so it won't load in Node regardless.

Reply:
No action items, just some background:

As I think I mentioned, when there are things that require DOM that have no direct Node analog, I tend to put them in `shells`, which is to say they are only for use by shells which provide the correct dependency (DOM in this case).

Because of the pluggable nature of services, there can be service implementations in various places, there is no requirement they all be in one place, nor should they be considered an atomic set (i.e. individual shells may want to load selected services, for dependency or other reasons).

Reply:
:+1: Thanks for the background.